### PR TITLE
#Milestone-2

### PR DESCRIPTION
**Pull request by MxoDevops**
#Add components
#Add UI
#Updated React App

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../calculator.css';
+
+class Calculator extends React.Component {
+  constructor() {
+    super();
+    this.MapCalculator = this.MapCalculator.bind(this);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```
Comment (ahmetbozaci) [inline src/components/calculator.js:10]: - [ ] Kindly don't disable the linter. If giving an error please fix it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @MxoDevops  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Kindly update your README file. 
You can use this [template](https://github.com/microverseinc/readme-template) 
and make sure there are no broken links(screenshot, license, etc.) and empty sections. 
Kindly change the readme template's sentence with your sentence.
image

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by MxoDevops** [comment]
Last week I submitted the 1st-day project of setting up the React app and it got approved. So now I opened this issue to resubmit the setup but I have already deleted the branch that was approved and merged.

**Comment by brytebee** [comment]
Okay. I have seen the [approved branch here.](https://github.com/MxoDevops/math-magicians/pull/1)

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @MxoDevops ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Professional documentation ✔️ 

**OPTIONAL**

- [ ] Please try to write meaningful and descriptive comments. One of the last comments you have is [Made changes](https://github.com/MxoDevops/math-magicians/pull/2/commits/a308bc461fc679cbc11ef240325cede6b290fa17) as you can guess this commit text is not descriptive nor informative

giphy

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**